wagtail/admin/static_src/wagtailadmin/scss/components/_tabs.scss

```diff
@@ -18,7 +18,7 @@
 
     a {
         @include transition(border-color 0.2s ease);
-        background-color: lighten($color-teal-darker, 3%);
+        background-color: $color-teal-darker;
```

Comment:
I'm unfortunately also finding this contrast to be insufficient, and finding the hover style to be be almost unnoticeable with the new colors. I'm not happy with any of the color options I'm thinking of at the moment, but it's also getting late here, so I'm hoping to have fresher eyes in the daytime.

This branch:
<img width="523" alt="image" src="https://user-images.githubusercontent.com/1051450/46268835-75c6dd00-c52c-11e8-9066-13539119f10d.png">
master:
<img width="520" alt="image" src="https://user-images.githubusercontent.com/1051450/46268854-9d1daa00-c52c-11e8-9d2a-d4fac255a8c2.png">

The middle tab has `:hover` applied on both

Reply:
@nmorduch I think this is better. Do you agree?

Reply:
Yes, I think this is much better!